app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -105,11 +105,40 @@ class MainFragment : Fragment() {
                     .navigate(MainFragmentDirections.actionMainFragmentToNewProfileFragment())
             }
         }
-
+        currentlyActiveProfile()
         setHasOptionsMenu(true)
         return binding.root
     }
 
+    private fun currentlyActiveProfile() {
+        StoreSession.init(context!!)
```

Comment:
@coder2699 The StoreSession initialization has to be done in the application class so that is initialized only once in the whole app lifecycle.

Reply:
@rob729 Actually, I didn't know about it, I will remove it from here.

Reply:
@rob729 I get it, I shall create an application class and initialize it there. Right?

Reply:
Yes.